spec/unit/block_score/actions/delete_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'faker'
+
+RSpec.describe BlockScore::Actions::Delete do
+  describe '#delete' do
+    subject(:candidate) { create(:candidate) }
+    before { candidate.delete }
+
+    it { is_expected.not_to be_persisted }
+    it { expect(candidate.deleted).to be true }
```

Comment:
Oh I know it's an attribute method. I was just saying that even though it's server state I would generally remap it onto an idiomatic method name rather than using the default accessor.